packages/client/hmi-client/src/components/asset/tera-asset.vue

```diff
@@ -62,7 +62,7 @@
 				</aside>
 			</header>
 		</template>
-		<section :style="stretchContentStyle">
+		<section class="overflow-hidden" :style="stretchContentStyle">
```

Comment:
That's not how it should be applied, but because this component is due for a clean up anyway, we will not bother.

Reply:
sorry i forgot to push a commit from my local git, I know you hate these classes 😈.  This is passed in as a prop in the meantime until this component is cleaned up